x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/flyout/index.tsx

```diff
@@ -82,6 +121,17 @@ export const EventFiltersFlyout: React.FC<EventFiltersFlyoutProps> = memo(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    // Initialize the store with the enriched event to allow render the form
+    useEffect(() => {
+      if (enrichedData) {
+        dispatch({
+          type: 'eventFiltersInitForm',
+          payload: { entry: getInitialExceptionFromEvent(enrichedData) },
+        });
+      }
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
can we try to remove this eslint disabled rule?

Reply:
done in https://github.com/elastic/kibana/pull/118683/commits/4b9723ea09468ab842b8eefa59899990faffd43f